app/controllers/surveys_controller.rb

```diff
@@ -1,4 +1,35 @@
 class SurveysController < ApplicationController
+
   def send_survey
+    respondent_ids = JSON.parse(params[:respondent_ids]).uniq
+
+    begin
+      survey_id = params[:id]
+      survey = Survey.find(survey_id)
+    rescue ActiveRecord::RecordNotFound
+      return render :json => {:error => :survey_not_found}, :status => :bad_request
+    end
+
+    if survey.first_question.nil?
+      return render :json => {:error => :survey_missing_first_question}, :status => :bad_request
+    end
+
+    if respondent_ids.empty?
```

Comment:
If the list of respondents to send to is empty, should we raise an error or just return 200 without initiating any sends? Agree that it's a strange use of the endpoint but doesn't seem like an error.